File: sdk-node/examples/features/archive.ts
/**
 * Soft-archive a feature. Idempotent.
 *
 * Run:
 *   npx tsx examples/features/archive.ts
 */

import { ThreeCommon } from '@3common/sdk'

const client = new ThreeCommon({
  apiKey: '3co_your_api_key_here',
})

const feature = await client.features.archive('feat_replace_with_real_id')

console.log(`archived ${feature.id ?? '?'} — active=${String(feature.active ?? false)}`)

File: sdk-python/examples/features/archive.py
"""Soft-archive a feature. Idempotent.

Run with: python examples/features/archive.py
"""

from __future__ import annotations

from threecommon import ThreeCommon


def main() -> None:
    with ThreeCommon(api_key="3co_your_api_key_here") as client:
        feature = client.features.archive("feat_replace_with_real_id")
        print(f"archived {feature.id} — active={feature.active}")


if __name__ == "__main__":
    main()
